### ISSUE TITLE
TextArea - DS site WCAG accessibility documentation

### ISSUE THREAD

**Issue by jcfilben** [grommet/hpe-design-system#4962]
Add Accessibility information to the TextArea page on the Design System site.

- [x] Assess which [WCAG rules](https://www.w3.org/WAI/WCAG22/quickref/?versions=2.2) are applicable to the component and if they are passing/failing.
- [x] Create a pull request on the Design System site to add this information to the component page. Follow the format here: https://github.com/grommet/hpe-design-system/pull/4675/files. The finished result should look something like this: https://design-system.hpe.design/components/menu?q=menu#wcag-compliance.
- [x] Create issues in the relevant repository (grommet, grommet-theme-hpe, etc.) for any rules where we are not meeting WCAG AA or if there is something observed that it feels like we should fix. We should create 1 issue per rule failure. Issues where we are not meeting AA should be tagged with 'bug'. All issues created through this process should have the 'accessibility' label and be added to the accessibility board.

The pull request should be up on Github within the first week of the sprint so that the second week of the sprint can be focused on review and polishing.

**Comment by CillianHPE** [comment]
@jcfilben See first pass of accessibility audit for TextArea **[here](https://hpe.sharepoint.com/:x:/r/teams/hpe-design/Shared%20Documents/HPE%20Design%20System/Accessibility/WCAG%20Rule%20Evaluation%20Template.xlsx?d=we975e19f32644f4cbd0a446f23f89277&csf=1&web=1&e=yagTne&nav=MTVfe0MwQzNDMUJCLTQ0NzEtNERCQy05NzlGLTk3MzM1RUVGNDVDQ30)**

**Comment by CillianHPE** [comment]
@jcfilben On a separate note I noticed that I can resize the textarea to a smaller size then the height of the first line of text.

I wonder if a minimum height value should be added to prevent the textarea from being resized to a smaller size than our form inputs.

![Image](https://github.com/user-attachments/assets/f40cdfd6-82ec-459d-9798-74c4d459ca61)

**Comment by jcfilben** [comment]
> @jcfilben On a separate note I noticed that I can resize the textarea to a smaller size then the height of the first line of text.
> 
> I wonder if a minimum height value should be added to prevent the textarea from being resized to a smaller size than our form inputs.

Good question, the implementer could add a min-height and min-width to the textarea but I think we could potentially consider an enhancement on the hpe theme to prevent textarea's from becoming too small.

**Comment by jcfilben** [comment]
Added some comments to the [excel file](https://hpe.sharepoint.com/:x:/r/teams/hpe-design/_layouts/15/Doc.aspx?sourcedoc=%7BE975E19F-3264-4F4C-BD0A-446F23F89277%7D&file=WCAG%20Rule%20Evaluation%20Template.xlsx&nav=MTVfe0MwQzNDMUJCLTQ0NzEtNERCQy05NzlGLTk3MzM1RUVGNDVDQ30&action=default&mobileredirect=true)

**Comment by CillianHPE** [comment]
**[Issues created](https://github.com/grommet/grommet/issues/7578)**
